### DISCUSSION TITLE
The number of node links affect the farm win？

### DISCUSSION THREAD

**Discussion by tracy32792** [Chia-Network/chia-blockchain#8138] [Support]
The number of node links affect the farm win？

**Comment by BrandtH22** [comment]
@tracy32792 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - while the number of peer nodes does not directly effect farming blocks, having more peers can lead to your node receiving the challenges from timelords and submitting blocks to timelords in a slightly faster manner as it is more likely to have a peer that either is a timelord or is close to a timelord.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
